articles/native-platforms/phonegap.md

```diff
@@ -6,6 +6,10 @@ hybrid: true
 image: //auth0.com/lib/platforms-collection/img/phonegap.png
 tags:
   - quickstart
+snippets:
+  dependancies: native-platforms/phonegap/dependancies
+  setup: native-platforms/jquery/setup
```

Comment:
I don't know if this was you or the elance person, but this is exactly right :).